analyzer_test.go

```diff
@@ -104,15 +104,13 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
 				h.AssertNil(t, json.Unmarshal(metadata, &appImageMetadata))
 			})
 
-			it("restores layer metadata and unsets the launch, build and cache flags", func() {
+			it("restores app and cache layer metadata without the launch, build and cache flags", func() {
 				_, err := analyzer.Analyze(image, testCache)
 				h.AssertNil(t, err)
 
 				unsetFlags := "[types]"
 				for _, data := range []struct{ name, want string }{
 					{"metadata.buildpack/launch.toml", "[metadata]\n  launch-key = \"launch-value\""},
-					{"metadata.buildpack/launch-build-cache.toml", "[metadata]\n  launch-build-cache-key = \"launch-build-cache-value\""},
```

Comment:
Are there any tests that check these files get written correctly when they do get written? I see `cache with inconsistent metadata exists` but it only tests for the absence of the files.  Or should this all be under a `cache does not exist` header?

Reply:
I think it's covered by: https://github.com/buildpacks/lifecycle/blob/8d93e588101b22f1e8d3badb9274ebb4bf0894bc/analyzer_test.go#L239-L240 (which remain intact)